components/desktop/partials/KnitImportantWidgetItem.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div v-if="show" class="knit-important-widget-item">
+    <span @click="hideMessage" class="knit-important-widget-item__message-hider"> x </span>
+    <h2 class="knit-important-widget-item__message">
+      {{ importantMessage }}
+    </h2>
+  </div>
+</template>
+
+<script>
+export default {
+  data () {
+    return {
+      show: true
+    }
+  },
+  methods: {
+    hideMessage (target) {
+      const widgetItem = target.path[1]
+      widgetItem.classList.add('swipe-hide')
+      // @TODO: Use template helper here + add some animation for display none
+      setTimeout(() => { this.show = false }, 700)
+    }
+  },
+  props: ['importantMessage']
+}
+</script>
+
+<style lang="scss">
+@import "assets/scss/imports.scss";
+
+.knit-important-widget-item {
+  position: relative;
+  &__message {
+    padding: 20px;
+    font-size: .85rem;
+    font-weight: normal;
+    color: $knit-important-widget-item-text-color;
+    background-color: $knit-important-widget-item-bg-color;
+    margin-bottom: $knit-default-gutters-width;
+    &:before {
+      font-weight: 900;
+      content: "Uwaga!";
```

Comment:
I think using a pseudoelement here is not good. It's not only a decorator, this text gives a value that message is important. This text won't be recognized by a screen reader, for example, so please write it in a text message.